contracts/balance/migration_test.go

```diff
@@ -60,7 +62,25 @@ func testMigrationFromDump(t *testing.T, d *dump.Reader) {
 	// try to update the contract
 	if notaryDisabled && prevPendingVotes {
 		c.CheckUpdateFail(t, "pending vote detected")
-		return
+		t.Fail()
```

Comment:
have u reached this? the change worth extension to tests of other contracts (and separation from the currentone)

Reply:
No. We don't have it in the current data set and we won't ever have it for obvious reasons, but I wanted to ensure that test doesn't silently "succeed". Can be reverted.